charts/owncloud/values.yaml

```diff
@@ -17,29 +17,543 @@ nameOverride: ""
+  # @default -- 20G
+  maxUpload: "20G"
+  # -- Enabled memory caching via memcached (see [documentation](https://doc.owncloud.com/server/latest/admin_manual/configuration/server/config_sample_php_parameters.html#memory-caching-backend-for-distributed-data)).
+  # @default -- false
+  memcachedEnabled: "false"
+  # -- Defines the hosts for memcached (see [documentation](https://doc.owncloud.com/server/latest/admin_manual/configuration/server/config_sample_php_parameters.html#define-server-details-for-memcached-servers-to-use-for-memory-caching)).
+  # @default -- memcached
+  memcachedHost: "memcached"
+  # -- Define connection options for memcached (see [documentation](https://doc.owncloud.com/server/latest/admin_manual/configuration/server/config_sample_php_parameters.html#define-connection-options-for-memcached)).
+  # @default -- 
+  memcachedOptions: ""
+  # -- Defines the ports for memcached (see [documentation](https://doc.owncloud.com/server/latest/admin_manual/configuration/server/config_sample_php_parameters.html#define-server-details-for-memcached-servers-to-use-for-memory-caching)).
+  # @default -- 11211
+  memcachedPort: "11211"
+  # -- Time to wait for a successful connection to the memcached service on container startup.
+  # @default -- 180
+  memcachedStartupTimeout: "180"
+  # -- Memory caching backend for locally stored data (see [documentation](https://doc.owncloud.com/server/latest/admin_manual/configuration/server/config_sample_php_parameters.html#memory-caching-backend-for-locally-stored-data)).
+  # @default -- ${OWNCLOUD_CACHING_CLASS:-\\OC\\Memcache\\APCu}
+  memcacheLocal: ""
+  # -- Define the memory caching backend for file locking (see [documentation](https://doc.owncloud.com/server/latest/admin_manual/configuration/server/config_sample_php_parameters.html#define-the-memory-caching-backend-for-file-locking)).
+  # @default -- 
+  memcacheLocking: ""
+  # -- Define the minimum supported ownCloud desktop client version (see [documentation](https://doc.owncloud.com/server/latest/admin_manual/configuration/server/config_sample_php_parameters.html#define-the-minimum-supported-owncloud-desktop-client-version)).
+  # @default -- 
+  minimumSupportedDesktopVersion: ""
+  mountFile: ""
+  # -- Define MySQL 3/4 byte character handling (see [documentation](https://doc.owncloud.com/server/latest/admin_manual/configuration/server/config_sample_php_parameters.html#define-mysql-34-byte-character-handling)).
+  # @default -- ${OWNCLOUD_UTF8MB4_ENABLED}
+  mysqlUtf8Mb4: ""
+  objectstore:
+    # -- Bucket name to store data.
+    # @default -- owncloud
+    bucket: "owncloud"
+    # -- Class to use for the objectstore. It is recommended to keep the default (see [documentation](https://doc.owncloud.com/server/latest/admin_manual/configuration/files/external_storage/s3_compatible_object_storage_as_primary.html)).
+    # @default -- OCA\Files_Primary_S3\S3Storage
+    class: "OCA\Files_Primary_S3\S3Storage"
```

Comment:
YAML needs `\\` to escape or single quotes around the entire value.

Reply:
```suggestion
    class: "OCA\\Files_Primary_S3\\S3Storage"
```